aries-site/src/examples/components/namevaluelist/Examples/NameValueListMultipleDefaultExample.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Box, Heading, NameValueList, NameValuePair, Tab, Tabs } from 'grommet';
+import { kubernetesData, serverData, tagsData } from '../data';
+
+export const NameValueListMultipleDefaultExample = () => {
+  const [index, setIndex] = React.useState();
+  const onActive = nextIndex => setIndex(nextIndex);
+  return (
+    <Box pad="small">
```

Comment:
Can we remove the Box / padding here? The example already seems to have plenty of padding

Reply:
yes I can remove